T14 enhance app UI

iliajamasb opened the pull request:
Hey,

In this branch, I updated a few UI features, making most pages look the same/have the same layout.
The main things that have been updated are the lessons pages, including the page to view lessons, request new ones, edit or delete them as well.

iliajamasb committed:
Improved nav-bar and bookings page UI

iliajamasb committed:
Brief changes to some form views

iliajamasb committed:
Merge branch 'main' into T14-Enhance-app-UI

iliajamasb committed:
Added UI aspects to lessons table

Ali-Alkhars commented on lessons/forms.py:
```diff
@@ -10,13 +10,13 @@ class Meta:
+            'availableTimes': "Choose a time",
+            'numberOfLessons': "Number of lessons desired",
+            'IntervalBetweenLessons': "Interval between lessons (In weeks)",
+            'DurationOfLesson': "Duration of lesson (In minutes)",
```
no need for '(in minutes)' cause they can't really enter numbers

Ali-Alkhars commented on lessons/templates/lesson_page.html:
```diff
@@ -1,60 +1,81 @@
+                    <tr>
+                        <th scope="col">student_id</th>
+                        <th scope="col">Available days</th>
+                        <th scope="col">Availble times</th>
```
A typo in 'Available'

Ali-Alkhars commented on lessons/templates/lesson_request.html:
```diff
@@ -1,15 +1,16 @@
-            {{form}}
-            <input type="submit" value="Submit lesson"> 
+            {% include 'partials/bootstrap_form.html' with form=form %}
+            <input type="submit" value="Request" class="btn btn-primary">
```
Would look nicer if you make the button in the centre instead of the left

Ali-Alkhars commented on lessons/templates/lesson_request_update.html:
```diff
@@ -1,7 +1,15 @@
-</form>
\ No newline at end of file
+    {% include 'partials/bootstrap_form.html' with form=form %}
+    <input type="submit" value="Update" class="btn btn-primary">
```
Would look nicer if you make the button in the centre instead of the left

Ali-Alkhars commented on lessons/templates/lesson_request_delete.html:
```diff
@@ -1,6 +1,14 @@
+{% extends 'partials/navbar.html' %}
+{% block content %}
+
```
This template you could add more meaning to it. For example, you could make it a 'ARE YOU SURE YOU WANT TO DELETE THIS REQUEST?' instead of just having a dull button

Ali-Alkhars reviewed (changes requested):
Amazing job honestly! Things look consistent and I really like the darker background when logged in. There are minor edits I requested in the comments. 

Also, there is this
<img width="1289" alt="Screenshot 2022-12-06 at 10 45 15 pm" src="https://user-images.githubusercontent.com/34315087/206040532-fbc65888-c9f1-42e0-a4ba-32b564b1efdf.png">
<img width="1289" alt="Screenshot 2022-12-06 at 10 44 15 pm" src="https://user-images.githubusercontent.com/34315087/206040565-1ff44cb1-9a35-4dd5-88ec-7bbad8a4171b.png">

It happens cause the requests above have less fields

iliajamasb committed:
Fixed UI bugs/spelling errors

Fixed table rows being inconsistent lengths
Added second confirmation when deleting request
And other general spelling mistakes

iliajamasb committed:
Redid migrations to push

chinwan34 committed:
Fixed migrations

chinwan34 committed:
Try again

chinwan34 committed:
Merged with main

chinwan34 committed:
Try

chinwan34 committed:
Try

Ali-Alkhars reviewed (approved):
Well done!
